docs/bart/1/bart_inst_guide/02_installing_bart_using_an_rpm_package.mdx

```diff
@@ -87,52 +85,77 @@ Upon successful installation, BART is installed in the `BART_HOME` directory:
 The installation includes the following files:
 
 <div class="table-with-scroll">
-<table>
-<colgroup>
-<col style="width: 31%" />
-<col style="width: 22%" />
-<col style="width: 45%" />
-</colgroup>
-<tbody>
-<tr class="odd">
-<td>File Name</td>
-<td><blockquote>
-<p>Location</p>
-</blockquote></td>
-<td>Description</td>
-</tr>
-<tr class="even">
-<td><code>bart</code></td>
-<td><code>&lt;BART_HOME&gt;/bin</code></td>
-<td>BART command line, executable program</td>
-</tr>
-<tr class="odd">
-<td><code>bart-scanner</code></td>
-<td><code>&lt;BART_HOME&gt;/bin</code></td>
-<td>BART WAL scanner program</td>
-</tr>
-<tr class="even">
-<td><code>bart.cfg.sample</code></td>
-<td><code>&lt;BART_HOME&gt;/etc</code></td>
-<td>Sample BART configuration file</td>
-</tr>
-<tr class="odd">
-<td><code>xlogreader_ident.so</code></td>
-<td><code>&lt;BART_HOME&gt;/lib</code></td>
-<td>Libraries supporting WAL versions</td>
-</tr>
-<tr class="even">
-<td><code>bart_license.txt</code></td>
-<td><code>&lt;BART_HOME&gt;</code></td>
-<td>License agreement</td>
-</tr>
-</tbody>
-</table>
+  <table class="table">
+    <colgroup>
+      <col style="width: 31%" />
+      <col style="width: 22%" />
+      <col style="width: 45%" />
+    </colgroup>
+    <tbody>
+      <tr class="odd">
```

Comment:
Since these have `odd` and `even` classed, I wonder if we should use `table-striped`?